CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### Features
+
+* show version with `-v` or `--version` option
+
```

Comment:
No need to touch `CHANGELOG.md`, it will be generated automatically based on your commit message.
```suggestion
```